src/api/v4/dataview/category.js

```diff
@@ -0,0 +1,240 @@
+var _ = require('underscore');
+var Base = require('./base');
+var constants = require('../constants');
+var CategoryDataviewModel = require('../../../dataviews/category-dataview-model');
+var CategoryFilter = require('../../../windshaft/filters/category');
+
+/**
+ * Category dataview object
+ *
+ * @param {carto.source.Base} source - The source where the dataview will fetch the data.
+ * @param {string} column - The column name to get the data.
+ * @param {object} options
+ * @param {carto.operation} options.operation - The operation to apply to the data.
+ * @param {string} options.operationColumn - The column name used in the operation.
+ *
+ * @constructor
+ * @extends carto.dataview.Base
+ * @memberof carto.dataview
+ * @api
+ */
+function Category (source, column, options) {
+  this._initialize(source, column, options);
+}
+
+Category.prototype = Object.create(Base.prototype);
+
+/**
+ * Set the dataview maxCategories
+ *
+ * @param  {string} maxCategories
+ * @return {carto.dataview.Category} this
+ * @api
+ */
+Category.prototype.setMaxCategories = function (maxCategories) {
+  this._checkMaxCategories(maxCategories);
+  this._options.maxCategories = maxCategories;
+  if (this._internalModel) {
+    this._internalModel.set('max_categories', maxCategories);
+  }
+  return this;
+};
+
+/**
+ * Return the current dataview maxCategories
+ *
+ * @return {string} Current dataview maxCategories
```

Comment:
`{string}` or `{number}`? Same applies to `Category.prototype.setMaxCategories`.

Reply:
:+1: